src/applications/vaos/reducers/newAppointment.js

```diff
@@ -168,10 +174,22 @@ export default function formReducer(state = initialState, action) {
         },
       };
     }
+    case FORM_RESET: {
+      return {
+        ...initialState,
+        systems: state.systems,
+        facilities: state.facilities,
+        clinics: state.clinics,
+        eligibility: state.eligibility,
+        pastAppointments: state.pastAppointments,
+        submitStatus: FETCH_STATUS.notStarted,
+      };
+    }
     case FORM_PAGE_CHANGE_STARTED: {
       return {
         ...state,
         pageChangeInProgress: true,
+        hasDataFetchingError: false,
```

Comment:
I'm not sure that 2 pages in a row ever use this flag, but if a user hits the back button, `FORM_PAGE_CHANGE_STARTED` doesn't get dispatched. Should we consider setting `hasDataFetchingError` to false on `FORM_PAGE_OPENED` as well?

Reply:
I think I should remove it from here entirely. I settled on hiding the back/continue buttons when you hit this error, since it's complicated to try to recover from the different errors without starting over entirely.